_episodes/01-introduction.md

```diff
@@ -51,19 +52,35 @@ A neural network is an artificial intelligence technique loosely based on the wa
 
 ![A diagram of a single artificial neuron](../fig/01_neuron.png){: width="600px"}
 
-Multiple neurons can be joined together by connecting the output of one to the input of another. These connections are associated with weights that determine the 'strength' of the connection, the weights are adjusted during training. In this way, the combination of neurons and connections describe a computational graph, an example can be seen in the image below. In most neural networks neurons are aggregated into layers. Signals travel from the input layer to the output layer, possibly through one or more intermediate layers called hidden layers. 
+Multiple neurons can be joined together by connecting the output of one to the input of another. These connections are associated with weights that determine the 'strength' of the connection, the weights are adjusted during training. In this way, the combination of neurons and connections describe a computational graph, an example can be seen in the image below. In most neural networks neurons are aggregated into layers. Signals travel from the input layer to the output layer, possibly through one or more intermediate layers called hidden layers.
 The image below shows an example of a neural network with three layers, each circle is a neuron, each line is an edge and the arrows indicate the direction data moves in.
 
 ![A diagram of a three layer neural network.](../fig/01_neural_net.png){: width="400px"}
 The image above is by Glosser.ca, CC BY-SA 3.0 <https://creativecommons.org/licenses/by-sa/3.0>, via Wikimedia Commons, [original source](https://commons.wikimedia.org/wiki/File:Colored_neural_network.svg)
 
 
+> ## Calculate the output for one neuron
+> Suppose we have
+> * Input: X = (0, 0.5, 1)
+> * Weights: W = (-1, -0.5, 0.5)
+> * Bias: b = 1
+> * Activation function _relu_: `f(x) = max(x, 0)`
+>
+> What is the output of the neuron?
+>
+> _Note: You can use whatever you like: brain only, pen&paper, Python, Excel..._
+>
+> > ## Solution
+> > The output is $f(b + \sum_{i=1}^n x_i w_i) = 1.25$
```

Comment:
```suggestion
> > input * weights = 0 * -1 + 0.5*-0.5 + 1*0.5 = 0.25
> > Add the bias: 0.25 + 1 = 1.25
> > Apply activation function: max(0, 1.25) = 1.25
> > Neuron output = 1.25
```
I suggest to just use this instead of the formula.
* Let's not commit something that is broken
* I think this is clearer for people that don't have a strong math background. It shows the different steps in calculating the output.

Reply:
Great!